File: examen-web-application/Services/RoomsService/adm/dto/RoomAdmDTO.cs
using examen_web_application.Services.RoomsService.dto;

namespace examen_web_application.Services.RoomsService.adm.dto
{
    public class RoomAdmDTO
    {
        public int ID { get; set; }
        public string Description { get; set; }
        public bool Available { get; set; }
        public double Price { get; set; }
        public RoomTypeDTO Type { get; set; }
        public string RoomNo { get; set; }
    }

}

File: examen-front-angular/src/app/services/room/adm/dto/roomAdmDTO.ts
import { RoomTypeDTO } from '../../dto/roomTypeDTO';

export class RoomAdmDTO {
    id: number;
    description: string;
    available: boolean;
    price: number;
    type: RoomTypeDTO;
    roomNo: string;

    constructor() {
        this.description = '';
        this.available = true;
        this.price = 100;
        this.type = { id: 0, name: 'Single' };
        this.roomNo = '101';
    }
}

